### PR TITLE
Resources initial draft

### PR DESCRIPTION
**Pull request by nupur-n**
Resources section with blank data. Will have to follow up w/ design about what they want the image to be, and get the resource guides from everyone. Right now it just points to google...

Questions: 
1. do we want some way to have it be clear when u hover over the card you click it?
2. is it bad to use an `<a>` tag... 
3. is there a better way to pass in the props than manually doing each one

image

### CODE REVIEW

File: src/styles/resources.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.resources {
+  &__card-label {
+    color: $mustard-yellow;
+    margin-left: 5px;
+    margin-bottom: 25px;
+  }
+
+  &__rounded-card {
```
Comment (feliciazhang) [inline src/styles/resources.scss:9]: does the cursor do the clicky hand thing when hovering? if not add cursor: pointer here

File: src/components/Resources.js [hunk 2]
```diff
@@ -1,5 +1,47 @@
 import React from 'react';
-import { Col, Container, Row } from 'react-bootstrap';
+import { Card, Col, Container, Row } from 'react-bootstrap';
+
+// Define the data needed for the resources cards
+const resources = [
```
Comment (feliciazhang) [inline src/components/Resources.js]: i think it'd be good to import this from a data file

**Review by feliciazhang — COMMENTED** [review]
re the qs, <a> tag is fine. for props you could just have a resource prop instead of splitting out the pieces but then you'd just have to decompose it in the card component anyway so doesn't really matter

File: src/styles/resources.scss (hunk 1 above)
Comment (nupur-n) [reply src/styles/resources.scss:9]: no it doesn't -- added

File: src/components/Resources.js (hunk 2 above)
Comment (nupur-n) [reply src/components/Resources.js]: good call, updated

**Review by feliciazhang — APPROVED** [review]

**Review by snwessel — APPROVED** [review]
This looks great! Sorry for the delay in reviewing!

**Pull request merged**